sematic/api/endpoints/tests/test_meta.py

```diff
@@ -0,0 +1,63 @@
+# Standard library
+from typing import Any, Dict, cast
+
+# Third-party
+import flask.testing
+import flask
+import pytest
+
+# Sematic
+from sematic.versions import CURRENT_VERSION, MIN_CLIENT_SERVER_SUPPORTS
+from sematic.api.tests.fixtures import (  # noqa: F401
+    mock_no_auth,
+    test_client,
+    mock_requests,
+    mock_user_settings,
+)
+from sematic.db.tests.fixtures import test_db, persisted_user  # noqa: F401
+import sematic.api.endpoints.meta  # noqa: F401
+from sematic.api.app import sematic_api  # noqa: F401
+from sematic.user_settings import SettingsVar
+
+
+@pytest.mark.parametrize(
+    "authenticate_config, expected_providers",
+    ((True, {"GOOGLE_OAUTH_CLIENT_ID": "ABC123"}), (False, {})),
+)
+def test_authenticate_endpoint(
+    authenticate_config: bool,
+    expected_providers: Dict[str, str],
+    test_client: flask.testing.FlaskClient,  # noqa: F811
+):
+    with mock_user_settings(
+        {
+            SettingsVar.SEMATIC_AUTHENTICATE: authenticate_config,
+            SettingsVar.GOOGLE_OAUTH_CLIENT_ID: "ABC123",
+        }
+    ):
+        response = test_client.get("/authenticate")
```

Comment:
This endpoint is not in `meta.py` IIUC. Why moving the test?

Reply:
oops! That was accidental